### ISSUE TITLE
How to get the event of the close button

### ISSUE THREAD

**Issue by qq869217311** [kamranahmedse/driver.js#340]
When clicking next or previous, if the route is skipped (the page is closed), onreset will also be triggered. The sequence is onnext > onreset | onprevious > onreset

**Comment by qq869217311** [comment]
How to get the event that users click to close

**Comment by qq869217311** [comment]
Onreset will be triggered due to page Jump,
Cannot directly process shutdown events in onreset

**Comment by debrianruhut** [comment]
Hi, I have the same issues, the button not working at all, the close button, next or prev button. Please help me.

**Comment by debrianruhut** [comment]
> Onreset will be triggered due to page Jump, Cannot directly process shutdown events in onreset

Hi, after hours of trial and error, I got the real problem of the button not working. You can try to remove temporary other .js after driver.js declaration. Try to comment out that .js and refresh the page.

**Comment by debrianruhut** [comment]
You can check the code that caused the problem here --> **click touchstart**:
```
  // This will hide sidebar when it's clicked outside of it
  $(document).on('click touchstart', function(e){
    e.stopPropagation();

    // closing left sidebar
    if($('body').hasClass('show-left')) {
      var targ = $(e.target).closest('.br-sideleft').length;
      if(!targ) {
        $('body').removeClass('show-left');
      }
    }

    // closing right sidebar
    if($('body').hasClass('show-right')) {
      var targ = $(e.target).closest('.br-sideright').length;
      if(!targ) {
        $('body').removeClass('show-right');
      }
    }
  });
```

**Comment by kamranahmedse** [comment]
We just released [v1.0](https://github.com/kamranahmedse/driver.js/releases/tag/1.0.3) of driver.js which has been rewritten from the ground up. There are more simplified hooks which give you more control over the popover and the whole lifecycle. Please have a [look at the docs](https://driverjs.com/) and do let me know if you face any issues.
